README.md

```diff
@@ -321,6 +322,19 @@ type loaders = {
 }
 ```
 
+For custom operation root types (queries not of type `Query`, or root type for other purpose), use the loader option `rootTypes`:
+
+- use a custom type name to override standard type
+- use a empty string to disable the custom standard type
+
+```js
+rootTypes: { 
```

Comment:
This was a bit hard to grok at first, but the information in the PR description was most helpful.

Reply:
Just to clarify, using a local schema you should use the following config

```json
  "loaders": {
    "GraphQLFileLoader": { // class name
      "module": "@graphql-tools/graphql-file-loader", // module name
      "options": { 
         "rootTypes": { 
           "query": "Root",
           "subscription": ""
         },
       }
    }
```

Reply:
oh, interesting!

Reply:
> I would also add the note about needing to install @graphql-tools/url-loader if you use a remote schema

That should already be covered by the current section Loaders: https://github.com/graphql-markdown/graphql-markdown#plugin-loaders

Reply:
Documentation updated
![Screenshot 2022-09-02 at 17 16 17](https://user-images.githubusercontent.com/324670/188180944-b6898396-08a0-473f-913b-53f08585caba.png)